resources/carrier/zh_CN/852.txt

```diff
@@ -0,0 +1,282 @@
+# Copyright (C) 2013 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Prefixes from http://www.ofca.gov.hk/mobile/en/industry_focus/telecommunications/portability/numbering/index.html
```

Comment:
One wikipedia page?

Just say something like this:

# Translations of en/852.txt from <wikipedia page> and in some cases the
carrier's own home page.

2016-06-30 9:38 GMT+02:00 Felix Yan <notifications@github.com>:

> In resources/carrier/zh_CN/852.txt
> <https://github.com/googlei18n/libphonenumber/pull/1181#discussion_r69084905>
> :
>
> > @@ -0,0 +1,282 @@
> > +# Copyright (C) 2013 The Libphonenumber Authors
> > +
> > +# Licensed under the Apache License, Version 2.0 (the "License");
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +
> > +# http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +# Unless required by applicable law or agreed to in writing, software
> > +# distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and
> > +# limitations under the License.
> > +
> > +# Prefixes from http://www.ofca.gov.hk/mobile/en/industry_focus/telecommunications/portability/numbering/index.html
>
> Most of the translations came from Wikipedia, and a few others are from
> their own homepage.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/googlei18n/libphonenumber/pull/1181/files/e64c1cfad1b8d757ad0ff422b477dec5bd63ab49#r69084905>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIjis4ldQvQInb9EArOVi8x8uFVeb91Sks5qQ3JbgaJpZM4JA2sK>
> .
>